modules/backend/formwidgets/relation/assets/less/relation.less

```
@@ -0,0 +1,78 @@
+@import "../../../../assets/less/core/boot.less";
+
+div[data-control="relation"] {
+
+    // Top widget controls
```

Stylelint (17.3.0)

[error] Unexpected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)